topology/probes/runc/state.json

```diff
@@ -0,0 +1,2958 @@
+{
```

Comment:
reduce the content to the minimum and include it in the test file please

Reply:
I have reduced context

regarding inlining the file as `parseHosts()` accepts a filename should we not leave `state.json` as a separate file?

Reply:
in the test file you can create a temp file ioutil.TempFile, it make no sense to pollute the repo with testing text files, specially not in tests/ subfolders